app/payment-request/get-correlation-id.js

```diff
@@ -0,0 +1,15 @@
+const db = require('../data')
+
+const getCorrelationId = async (paymentRequestId) => {
+  const correlationId = await db.paymentRequest.findOne({
```

Comment:
Maybe we should name this variable `paymentRequest` rather than `correlationId` so it's more clear what we're returning.  Or alternatively if the caller only needs a correlationId then just return that rather than object with only a correlationId to simplify it?

Reply:
updated as suggested to only return paymentRequest.correlationId